wiiu/vino/vino.ts

```diff
@@ -398,58 +404,58 @@ export default interface VinoAPI {
 	/**
 	 * Opens the browser to the given URL
 	 * @param url - URL to jump to
-	 * @param unk - Unknown
+	 * @param flag - (true/false)
 	 */
-	jumpToBrowser: (url: string, unk: boolean) => void;
+	jumpToBrowser: (url: string, flag: boolean) => void;
 
 	/**
 	 * Opens the eShop
-	 * @param unk1 - Unknown. Title ID?
-	 * @param unk2 - Unknown
+	 * @param titleId - Title ID
+	 * @param flag - (true/false)
 	 */
-	jumpToEShop: (unk1: string, unk2: boolean) => void;
+	jumpToEShop: (titleId: string, flag: boolean) => void;
 
 	/**
 	 * Opens Miiverse
-	 * @param unk - Unknown
+	 * @param flag - (true/false)
 	 */
-	jumpToMiiverse: (unk: boolean) => void;
+	jumpToMiiverse: (flag: boolean) => void;
 
 	/**
 	 * Opens Miiverse to a specific post
 	 * @param postID - Post ID
-	 * @param unk - Unknown
+	 * @param flag - (true/false)
 	 */
-	jumpToMiiversePostId: (postID: string, unk: boolean) => void;
+	jumpToMiiversePostId: (postID: string, flag: boolean) => void;
 
 	/**
-	 * Unknown
-	 * @param unk1 - Unknown
-	 * @param unk2 - Unknown
-	 * @param unk3 - Unknown
+	 * Jumps to a streaming app (Netflix, Hulu or Amazon) with an URL param
+	 * @param url - Streaming program ID?
+	 * @param titleID - Title ID of the streaming app
+	 * @param flag - (true/false)
 	 */
-	jumpToVod: (unk1: string, unk2: string, unk3: boolean) => void;
+	jumpToVod: (url: string, titleID: string, flag: boolean) => void;
 
 	/**
 	 * Open an installed title
 	 * @param titleID - Titles title ID (hex)
-	 * @param unk - Unknown
+	 * @param flag - (true/false)
```

Comment:
It was clear this is a true/false flag already. This was left as `unk` because the _use_ of the flag is unknown. This adds no extra details